scripts/stats.sql

```diff
@@ -0,0 +1,77 @@
+\o ./out.txt
+
+select current_timestamp "Date de l'extraction";
+
+-- Nb utilisateurs DGCCRF
+select count(*) "Nb utilisateurs DGCCRF" from users where role = 'DGCCRF';
+
+-- Nb utilisateurs DGCCRF connectés dans les 60 derniers jours
+select count(distinct(u.id)) "Nb utilisateurs DGCCRF connectés dans les 60 derniers jours"
+from users u, auth_attempts a
+where u.role = 'DGCCRF'
+and u.email = a.login
+and a.timestamp >= 'now'::timestamp - '60 days'::interval;
+
+-- Nb abonnements DGCCRF
+select count(*) "Nb abonnements DGCCRF"
+from subscriptions s, users u
+where u.role = 'DGCCRF'
+and u.id = s.user_id;
```

Comment:
À quoi sert cette ligne si on sélectionne uniquement le compte total ? On veut `user_id IS NOT NULL` ?

Reply:
C'est pour ne pas comptabiliser les abonnements avec le rôle `ADMIN` ou des abonnements directement par mails pour les boites institutionnelles donc sans `user_id`.
Est-ce que c'était le sens de ta question ?

Reply:
OK, j'avais zappé qu'il pouvait y avoir des abos non rattaché à un user DGCCRF